contexts/AuthContext.tsx

```diff
@@ -5,8 +5,8 @@ import React, {
   useContext,
   ReactNode,
 } from "react"
-import AsyncStorage from "@react-native-async-storage/async-storage"
 import * as SplashScreen from "expo-splash-screen"
+import AsyncStorage from "@react-native-async-storage/async-storage"
```

Comment:
this change is unnecessary

Reply:
what do you mean, the AsyncStorage is being used

Reply:
yes but it was already there you did not have to move it